### PR TITLE
LPS-180427 Style changes to modified paddings for the Add Panel for Widget Pages

### PR DESCRIPTION
**Pull request by Stephy6**
Motivation
=======
Problem this work solves are the styles of the Add Panel for Widget Pages.

Proposed Solution
========
What we did to solve this problem is modify the padding of the by removing the padding for the panel and adding the correct one for the TabPane outside of the panel. Also we modified the padding of the tabs.

Steps to Verify
========
1. Create a Widget Page and edit it
2. Click over the  button on the Control Menu navigation bar to open the add panel
3. Check that the view of the panel is the correct one (look added screenshot)

Screenshots 
========
Screenshot 2023-03-31 at 11 49 26

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Review by p2kmgcl — CHANGES_REQUESTED** [review]

File: modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/js/AddPanel.js [hunk 1]
```diff
@@ -198,12 +198,12 @@ const AddPanel = ({
...
+		rtl,
+	});
+
 	return (
-		<div
-			className={classNames('sidebar-body__add-panel', {
-				rtl,
-			})}
-		>
+		<div className={`${addPanelClassName} p-0`}>
```
[the remark this answers is not in the document]
Comment (Stephy6) [reply modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/js/AddPanel.js:206]: Understood @p2kmgcl , thank you!

**Review by p2kmgcl — APPROVED** [review]

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by p2kmgcl** [comment]
(waiting until CI is fixed)

**Comment by victorg1991** [comment]
ci:forward

**Pull request closed without merging**